build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CheckApiCompatibility.cs

```diff
@@ -31,6 +31,7 @@ public sealed class CheckApiCompatibility : Task
 			{ "v12.0", "v11.0" },
 			{ "v12.1", "v12.0" },
 			{ "v13.0", "v12.1" },
+			{ "v13.0.99", "v13.0" },
```

Comment:
Does this task *work* in a .NET Android world order?

Reply:
Apparently not!  😁 

It appears we only run ApiCompat against the *stable* API platform on .NET.  This is something that will need to be fixed, but will be much easier once Classic has been removed from `main`.  I propose we prioritize this work for later.  I ran it manually locally, so the breakages included in this PR are correct, but CI will not be able to verify them until then.

At the very worst, the PR in ~June that marks API-34 stable will check it with today's ApiCompat.